### PR TITLE
JENKINS-62066: Changed helptext fontsize to 14px 0.875rem

### PR DESCRIPTION
**Pull request by TapanGujjar**
See [JENKINS-62066](https://issues.jenkins-ci.org/browse/JENKINS-62066).

### Proposed changelog entries

* Entry 1: Issue, Human-readable Text
Changed the size of help text in job config forms to 14 px or 0.875rem.

### Proposed upgrade guidelines

[None]

### Submitter checklist

- [x] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

File: war/src/main/less/base/style.less [hunk 1]
```diff
@@ -789,6 +789,7 @@ LABEL.attach-previous {
   padding: 1em;
   margin-bottom: 1em;
   word-break: break-word;
+  font-size: @font-size-sm !important;
```
Comment (fqueiruga) [inline war/src/main/less/base/style.less:792]: We shouldn't be using important on base styles, why is this needed? Which places needs overriding?

**Review by fqueiruga — CHANGES_REQUESTED** [review]

**Comment by oleg-nenashev** [comment]
@TapanGujjar Hi. Did you have a chance to address the feedback?

**Comment by alecharp** [comment]
@TapanGujjar I'm proposing this PR for close. Please let us know if you are able to work on it, if not, we will close this.

**Comment by Vishalghyv** [comment]
This issue can be closed because the change requested are fixed see #5163

**Pull request closed without merging**